Bundle/FormBundle/Resources/public/js/link.js

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
Not sure what it is supposed to do because since the LinkType was refactored (https://github.com/Victoire/victoire/commit/d791815c2e3b7846ab1430c4903b0100a3e2d19a), we don't need any script for this